targets/targets.json

```diff
@@ -571,6 +571,20 @@
         "release_versions": ["2", "5"],
         "device_name": "MKW41Z512xxx4"
     },
+    "RO359B": {
```

Comment:
You got the file structure right but not json. A good example is ``MCU_K22F512`` - look at K22F how it inherits